python/ray/autoscaler/_private/_azure/azure-vm-template.json

```diff
@@ -44,6 +44,13 @@
                 "description": "The version of the VM image"
             }
         },
+        "diskSizeGB": {
+            "type": "int",
+            "defaultValue": 64,
```

Comment:
> however, @VikParuchuri modify your PR so the the json passes a null if the user does not set the disk size since the default sizes seem to creep up over time -- current defaults are at 150GB.

We should address this comment?